integration-tests/emulated/src/mock/para_chains.rs

```diff
@@ -29,19 +30,12 @@ pub mod spiritnet {
 
 	use spiritnet_runtime::{
 		BalancesConfig, ParachainInfoConfig, PolkadotXcmConfig, RuntimeGenesisConfig, SessionConfig, SessionKeys,
-		SystemConfig, WASM_BINARY,
 	};
 
-	pub const PARA_ID: u32 = 2_000;
+	pub const PARA_ID: u32 = 2_001;
```

Comment:
?

Reply:
Peregrine and Spiritnet are running on the same network. Para id 2000 was already occupied by Spiritnet.